docs/install/grafana-setup.rst

```diff
@@ -158,12 +165,12 @@ connection is successful.
 
 .. _grafana-import-dashboard-file:
 
-Import the Omniperf dashboard file
-----------------------------------
+Import the ROCm Compute Profiler dashboard file
+-----------------------------------------------
 
 From the **Create** → **Import** page, upload the dashboard file,
 ``/dashboards/Omniperf_v{__VERSION__}_pub.json`` from the
```

Comment:
Should this still reference Omniperf? Might not be updated.

Reply:
@xuchen-amd , are you renaming these files as a part of a subsequent change or will they keep they're name for now?

Reply:
patch PR created: https://github.com/ROCm/rocprofiler-compute/pull/469